kernel/src/tbfheader.rs

```diff
@@ -128,6 +132,31 @@ crate struct TbfHeaderV2WriteableFlashRegion {
     writeable_flash_region_size: u32,
 }
 
+/// Optional fixed addresses for flash and RAM for this process.
+///
+/// If a process is compiled for a specific address this header entry lets the
+/// kernel know what those addresses are.
+///
+/// If this header is omitted the kernel will assume that the process is
+/// position-independent and can be loaded at any (reasonably aligned) flash
+/// address and can be given any (reasonable aligned) memory segment.
+///
+/// If this header is included, the kernel will check these values when setting
+/// up the process. If a process wants to set one fixed address but not the other, the unused one
+/// can be set to 0xFFFFFFFF.
+#[derive(Clone, Copy, Debug, Default)]
+crate struct TbfHeaderV2FixedAddresses {
+    /// The absolute address of the start of RAM that the process expects. For
+    /// example, if the process was linked with a RAM region of length 4 kB
+    /// starting at address `0x00023000`, then this would be set to
+    /// `0x00023000`.
```

Comment:
```suggestion
    /// example, if the process was linked with a RAM region starting at 
    /// address `0x00023000`, then this would be set to `0x00023000`.
```
What does length have to do with anything here?

Reply:
Nothing, which was my point, but yeah probably makes it more confusing.